.github/workflows/workflow.yml

```diff
@@ -98,6 +98,23 @@ jobs:
         echo "$GITCOOKIE_SH" | bash
         bin/lint --verbose
 
+  go_fmt:
+    name: Go Format
+    runs-on: ubuntu-18.04
+    container:
+      image: golang:1.12.9
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v1
+    - name: Format
+      env:
+        GITCOOKIE_SH: ${{ secrets.GITCOOKIE_SH }}
+        # prevent OOM
+        GOGC: 20
+      run: |
+        echo "$GITCOOKIE_SH" | bash
```

Comment:
i don't think `GITCOOKIE_SH` or `GOGC` should be necessary. `GITCOOKIE_SH` mitigates rate-limiting issues when downloading go dependencies, and `GOGC` is a special tuning to prevent golangci-lint from OOM'ing.

Reply:
Would we not need to potentially worry about downloading go dependencies when running `go install -mod=readonly golang.org/x/tools/cmd/goimports`?

Reply:
Probably less a concern as I wouldn't expect that install command to download many deps, but yeah I guess it doesn't hurt to leave `GITCOOKIE_SH` in for now. FWIW we should be able to remove this in Go 1.13.